### ISSUE TITLE
Implement SVM

### ISSUE THREAD

**Issue by jacksonneal** [jacksonneal/gesture_recognition#3]

**Comment by jacksonneal** [comment]
- [ ] Implement model from scratch.
- [ ] Evaluate model including accuracy, precision, and recall.
- [ ] Experiment with model hyperparameters, considering overfitting, bias, variance, etc.
- [ ] Identify low performance examples and investigate cause.
- [ ] Consider primary component analysis, input skews.
- [ ] Compare performance with that of out of box model (sklearn).
- [ ] Prepare visuals highlighting findings and comparisons.
